packages/ci/build-prod-images.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
```

Comment:
The first line of a bash script should be at least `set -e`, so the script fails if any program exists with non-zero value. 
Background: https://wizardzines.com/comics/bash-errors/